.gitignore

```diff
@@ -1,5 +1,4 @@
 # Binaries for programs and plugins
-output
```

Comment:
Not sure right now if ignoring "output" is needed now.  There is a helpers/output package in this PR which breaks unless we make this ignore more specific, or rename the helpers/output package